### PR TITLE
Update attributes and attributes.values

### LINKED ISSUE #18: Update attributes and attributes.values
Simplify and consolidate the existing attributes so that these are the only top level attributes

- Gender
- Age group
- Shoes sizes
- Bra sizes
- Diaper sizes
- Clothing sizes

### PR DESCRIPTION
**Pull request by naphets123**
Simplified attributes to the ones stated in the issue + condition, fixes #18

### CODE REVIEW

File: data/product.attribute.csv [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
 product_attribute_DS,Diaper Sizes,TRUE
-product_attribute_MS,Mixed Sizes,FALSE
-product_attribute_O,Onesize,FALSE
 product_attribute_CS,Clothing Sizes,TRUE
-product_attribute_CSS,Children Shoe Sizes,TRUE
-product_attribute_FSS,Female Shoe Sizes,TRUE
-product_attribute_MSS,Male Shoe Sizes,TRUE
+product_attribute_SS, Shoe Sizes,TRUE
 product_attribute_G,Gender,TRUE
 product_attribute_AG,Age_Group,TRUE
```
Comment (jamescrowley) [inline data/product.attribute.csv]: @naphets123 hope you don't mind me jumping in, but is there a reason for keeping the underscore in Age group vs the others?

File: data/product.attribute.value.csv [hunk 2]
```diff
@@ -1,75 +1,66 @@
...
-product_attribute_value_1315y,boxwise_WMS.product_attribute_CbA,13-15 years
 product_attribute_value_Size1,boxwise_WMS.product_attribute_DS,Size 1
 product_attribute_value_Size2,boxwise_WMS.product_attribute_DS,Size 2 (3-5 kg)
 product_attribute_value_Size3,boxwise_WMS.product_attribute_DS,Size 3 (6-9 kg)
 product_attribute_value_Size4,boxwise_WMS.product_attribute_DS,Size 4 (10-18 kg)
 product_attribute_value_Size5,boxwise_WMS.product_attribute_DS,Size 5
 product_attribute_value_Size6,boxwise_WMS.product_attribute_DS,Size 6
-product_attribute_value_Mixed,boxwise_WMS.product_attribute_MS,Mixed
-product_attribute_value_Onesize,boxwise_WMS.product_attribute_O,Onesize
+product_attribute_value_Onesize,boxwise_WMS.product_attribute_CS,Onesize
```
Comment (jamescrowley) [inline data/product.attribute.value.csv]: Likewise, perhaps 'One size', given there are already spaces in other attribute names?

File: data/product.attribute.csv [hunk 3]
```diff
@@ -1,14 +1,8 @@
...
 product_attribute_BS,Bra Sizes,TRUE
-product_attribute_CbA,Children by Age,TRUE
 product_attribute_DS,Diaper Sizes,TRUE
-product_attribute_MS,Mixed Sizes,FALSE
-product_attribute_O,Onesize,FALSE
 product_attribute_CS,Clothing Sizes,TRUE
-product_attribute_CSS,Children Shoe Sizes,TRUE
-product_attribute_FSS,Female Shoe Sizes,TRUE
-product_attribute_MSS,Male Shoe Sizes,TRUE
+product_attribute_SS, Shoe Sizes,TRUE
```
Comment (jamescrowley) [inline data/product.attribute.csv]: Not sure if it matters for actual import, but I'm guessing the space here is a typo?

File: data/product.attribute.csv (hunk 1 above)
Comment (HaGuesto) [reply data/product.attribute.csv]: +1

File: data/product.attribute.value.csv (hunk 2 above)
Comment (HaGuesto) [reply data/product.attribute.value.csv]: +1

**Comment by jamescrowley** [comment]
Took the liberty of updating the description so it references the ticket (GH doesn't link them based on the title)

**Comment by HaGuesto** [comment]
@naphets123 we close this branch since we have optimized the categories for Pampiraiki now.

**Pull request closed without merging**